server/src/test/java/org/jetbrains/bsp/bazel/server/diagnostics/DiagnosticsServiceTest.kt

```diff
@@ -223,7 +223,7 @@ class DiagnosticsServiceTest {
ains/bsp/bazel/projectview/parser:parser_mock_test_impl //executioncontext/projectview:parser //server/src/main/java/org/jetbrains/bsp/bazel:bsp-install //executioncontext/workspacecontext/src/test/java/org/jetbrains/bsp/bazel/workspacecontext:WorkspaceContextTest //executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/parser/splitter:ProjectViewRawSectionTest //e2e/src/main/java/org/jetbrains/bsp/bazel:BazelBspEntireRepositoryImportTest //e2e:LocalJdkTest //executioncontext/installationcontext/src/test/java/org/jetbrains/bsp/bazel/installationcontext:InstallationContextTest //server/src/main/java/org/jetbrains/bsp/bazel/server:bsp-run //executioncontext/projectview:model //commons:commons //executioncontext/workspacecontext/src/main/java/org/jetbrains/bsp/bazel/workspacecontext/entries/mappers:mappers //server/src/main/java/org/jetbrains/bsp/bazel/server/sync/proto:bsp_target_info_java_proto //executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/parser/sections:ProjectViewSingletonSectionParserTest //e2e/src/main/java/org/jetbrains/bsp/bazel:BazelBspJava8ProjectTest //executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/model/sections:sections //logger/src/main/java/org/jetbrains/bsp/bazel/logger:logger //server/src/test/java/org/jetbrains/bsp/bazel/server/bsp/utils:InternalAspectsResolverTest //executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/parser/splitter:ProjectViewSectionSplitterTest //executioncontext/workspacecontext/src/main/java/org/jetbrains/bsp/bazel/workspacecontext:workspacecontext //logger:logger //server/src/main/java/org/jetbrains/bsp/bazel/server/diagnostics/parser:parser //server/src/test/java/org/jetbrains/bsp/bazel/server/diagnostics:diagnostics //executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/model:model //install/src/main/java/org/jetbrains/bsp/bazel/install/cli:cli //executioncontext/workspacecontext/src/test/java/org/jetbrains/bsp/bazel/workspacecontext/entries/mappers:WorkspaceContextTargetsEntityMapperTest //server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/utils:utils //executioncontext/installationcontext/src/main/java/org/jetbrains/bsp/bazel/installationcontext:installationcontext //server/src/test/java/org/jetbrains/bsp/bazel/server/bsp/utils:SourceRootGuesserTest //install:install //server/src/test/java/org/jetbrains/bsp/bazel/server/sync/dependencytree:DependencyTreeTest //server/src/main/java/org/jetbrains/bsp/bazel/server/sync/proto:bsp_target_info_proto //server/src/main/java/org/jetbrains/bsp/bazel/server/sync:sync //executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/parser:parser //server/src/main/java/org/jetbrains/bsp/bazel:bsp-lib //bazelrunner/src/main/java/org/jetbrains/bsp/bazel/bazelrunner/params:params //commons/src/test/java/org/jetbrains/bsp/bazel/commons:FormatTest //e2e:all //e2e/src/main/java/org/jetbrains/bsp/bazel/base:base
```

Comment:
please undo the formatting

Reply:
I didn't do formatting here

Reply:
did u change something here? if not just revert the changes